test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1

```diff
@@ -803,6 +886,23 @@ Describe "Invoke-WebRequest tests" -Tags "Feature" {
         $response.Content.UserAgent | Should Match $Pattern
     }
 
+    It "Verifies Invoke-WebRequest Certificate Authentication Fails without -Certificate"{
+        $command = 'Invoke-WebRequest https://prod.idrix.eu/secure/'
```

Comment:
It would definitely make these slow tests and I think feature tests.  There are some tests that use docker here:  https://github.com/PowerShell/PowerShell/tree/master/test/docker/networktest   But those tests, are meant to be a different suite.

I think @iSazonov  might be right.  I think option 3 might be the best option.  Mark the tests as pending, and file an issue.  If you are still willing to work on trying to fix the tests, I'd love to work with you.  Assuming @adityapatwardhan (the maintainer for the PR) agrees.